po/common/common.pot

```diff
@@ -5747,6 +5747,14 @@ msgctxt "ecoscore_warning_international"
 msgid "The Eco-Score was initially developped for France and it is being extended to other European countries. The Eco-Score formula is subject to change as it is regularly improved to make it more precise and better suited to each country."
 msgstr "The Eco-Score was initially developped for France and it is being extended to other European countries. The Eco-Score formula is subject to change as it is regularly improved to make it more precise and better suited to each country."
 
+msgctxt "ecoscore_warning_transportation"
```

Comment:
I'm thinking that it would be better to have different messages for the 2 cases: not having a country selected, and having a country selected that we don't support yet.

The messages could be something like:

short: The impact of transportation is not included.

long on world: Select a country in order to include the impact of transportation.
long on a not supported country: The impact of transportation to your country is currently unknown.

Reply:
Do we need to say "...the **full** impact of transportation..." as some impacts are still shown even for world?

Reply:
Sounds good, we can say "the full impact"